BlueprintUI/Sources/Element/UIViewElement.swift

```diff
@@ -96,7 +105,15 @@ public extension UIViewElement {
     
     /// Defer to the reused measurement view to provide the size of the element.
     var content: ElementContent {
-        ElementContent {
+        let key : MeasurementCachingKey? = {
+            if let key = self.measurementCacheKey {
+                return .init(type: Self.self, input: key)
+            } else {
+                return nil
+            }
+        }()
```

Comment:
This looks like a job for `Optional.map`:

```swift
let key = self.measurementCacheKey.map { key in
  MeasurementCachingKey(type: Self.self, input: key)
}
```

Reply:
Truuuuu